Source/WebKit/UIProcess/Cocoa/WebPageProxyCocoa.mm

```diff
@@ -260,6 +260,16 @@ static bool exceedsRenderTreeSizeSizeThreshold(uint64_t thresholdSize, uint64_t
 #if ENABLE(CONTENT_FILTERING)
 void WebPageProxy::contentFilterDidBlockLoadForFrame(IPC::Connection& connection, const WebCore::ContentFilterUnblockHandler& unblockHandler, FrameIdentifier frameID)
 {
+#if HAVE(PARENTAL_CONTROLS_WITH_UNBLOCK_HANDLER)
+    bool usesWebContentRestrictions = false;
+#if HAVE(WEBCONTENTRESTRICTIONS)
+    usesWebContentRestrictions = protectedPreferences()->usesWebContentRestrictionsForFilter();
+#else
+    if (usesWebContentRestrictions)
+        MESSAGE_CHECK(unblockHandler.webFilterEvaluatorData().isEmpty(), connection);
+#endif
```

Comment:
It seems we are never doing the `MESSAGE_CHECK` here if `HAVE(WEBCONTENTRESTRICTIONS)` is 1, is that expected?

Reply:
ah nice catch! The `#else` should be `#endif`; will update!